templates/operator/templates/stateful-set.yaml.template

```diff
@@ -330,7 +338,9 @@ spec:
             - bash
             - -c
           args:
-            - {{ if ne $.Params.JMX_LOCAL_ONLY "true" }}
+            - /etc/cassandra/generate-cassandra-yaml.sh;
+              {{ if ne $.Params.JMX_LOCAL_ONLY "true" }}
+              /etc/cassandra/generate-nodetool-ssl-properties.sh &&
               cp /nodetool-ssl-properties/nodetool-ssl.properties /home/cassandra/.cassandra/nodetool-ssl.properties;
```

Comment:
```suggestion
              cp /etc/cassandra/nodetool-ssl.properties /home/cassandra/.cassandra/nodetool-ssl.properties;
```

Reply:
Good catch! I'm was watching this very test in the CI right now. It will fail.